starter-code/pizza.js

```diff
@@ -1 +1,78 @@
 // Write your Pizza Builder JavaScript in this file.
+
+var btnPep = $('.btn-pepperonni');
+var btnMush = $('.btn-mushrooms');
+var btnPeppers = $('.btn-green-peppers');
+var btnSauce = $('.btn-sauce');
+var btnCrust = $('.btn-crust');
+
+
+$(document).ready(function() {
+  $('.pep').hide();
+  $('.mushroom').hide();
+  $('.green-pepper').hide();
+  $('.sauce-white').removeClass('sauce-white');
+  btnPeppers.toggleClass('active');
+  btnMush.toggleClass('active');
+  btnPep.toggleClass('active');
+  btnSauce.toggleClass('active');
+  btnCrust.toggleClass('active');
+  $(".price li:nth-child(5)").css("display", "none");
+  $(".price li:nth-child(1)").css("display", "none");
+  $(".price li:nth-child(2)").css("display", "none");
+  $(".price li:nth-child(3)").css("display", "none");
+  $(".price li:nth-child(4)").css("display", "none");
+  $('.crust').toggleClass('crust-gluten-free');
```

Comment:
Esta función te evita escribir la de los 5 botones. Puede tener algún error, pero solo por que os hagáis una idea de como reutilizar código.

function button (button,referenceClass, toogleClass, position) {
button.click(function(e) {
    $(referenceClass).toggle(toogleClass);
    button.toggleClass('active');
    $(".price li:nth-child(position)").removeAttr( "style" );
    var totalPay = updatePrice($(".price > strong").text(), 1,$(this).hasClass("active"));
    $(".price > strong").text(totalPay);
}